src/cascadia/TerminalApp/App.cpp

```diff
@@ -690,7 +690,7 @@ namespace winrt::TerminalApp::implementation
     //      Negative values of `delta` will move the view up by one page, and positive values
     //      will move the viewport down by one page.
     // Arguments:
-    // - delta: The direction to move the view relative to the current viewport(it 
+    // - delta: The direction to move the view relative to the current viewport(it
     //      is clamped between -1 and 1)

```

Comment:
```suggestion
    //      (it is clamped between -1 and 1)
```

Reply:
Actually it results from auto trimming of  trailing space. Not quite sure should i change unrelated lines, but i will follow your comment first.